lib/features/children/add_member/cubit/add_member_cubit.dart

```diff
@@ -143,36 +143,49 @@ class AddMemberCubit extends Cubit<AddMemberState> {
 
     emit(state.copyWith(status: AddMemberStateStatus.loading));
     try {
-      await _addMemberRepository.addMembers(members);
-
-      emit(state.copyWith(status: AddMemberStateStatus.success));
-
-      unawaited(
-        AnalyticsHelper.logEvent(
-          eventName: AmplitudeEvents.vpcAccepted,
-        ),
-      );
+      final membersAddedSuccessfully =
+          await _addMemberRepository.addMembers(members);
+
+      if (membersAddedSuccessfully) {
+        emit(state.copyWith(status: AddMemberStateStatus.success));
+        unawaited(
+          AnalyticsHelper.logEvent(
+            eventName: AmplitudeEvents.vpcAccepted,
+          ),
+        );
 
-      unawaited(
-        AnalyticsHelper.logEvent(
-          eventName: AmplitudeEvents.memberCreatedSuccesfully,
-          eventProperties: {
-            'nrOfMembers': members.length,
-            'memberNames': memberNames,
-          },
-        ),
-      );
+        unawaited(
+          AnalyticsHelper.logEvent(
+            eventName: AmplitudeEvents.memberCreatedSuccesfully,
+            eventProperties: {
+              'nrOfMembers': members.length,
+              'memberNames': memberNames,
+            },
+          ),
+        );
+      }
     } catch (error) {
       await LoggingInfo.instance.error(error.toString());
-      unawaited(
-        AnalyticsHelper.logEvent(
-          eventName: AmplitudeEvents.failedToCreateMember,
```

Comment:
I think we need to send the event not only for VPC_NOT_SUCCESSFUL

Reply:
I'll change the amplitude event to "failed_to_get_vpc", and put the more generic failedToCreateMember event to the catch all error.